misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -174,7 +183,10 @@ abstract class AnyAssertionsSpec(
                         expectSubject.isNoneOfFun(1, arrayOf(2))
                     }.toThrow<AssertionError> {
                         message {
-                            contains(IS_NONE_OF.getDefault(), "${listBulletPoint}1")
+                            containsRegex(
+                                "\\Q$rootBulletPoint${IS_NONE_OF.getDefault()}\\E:.*$separator" +
+                                    "$indentBulletPoint${listBulletPoint}1"
+                            )
```

Comment:
This is super hard to parse for a human.  Some time ago, I wouldn’t even have known what `\\Q` and `\\E` do.

Can we split this up into multiple, simpler `contains` statement? That would be easier to read both in code and in reporting.

Reply:
I really want to make sure the formatting is correct, hence we need to stick to this.